### ISSUE TITLE
lambdamenu.asi - this ASI plugin does not claim to support game build 2372

### ISSUE THREAD

**Issue by JamesSc0tt** [citizenfx/project-lambdamenu#7]
Since the requirement of FiveM to explicitly advertise game client supported versions, the Lambda menu will not load:
```
lambdamenu.asi - this ASI plugin does not claim to support game build 2372. If you have access to its source code, add FX_ASI_BUILD 2372 dummy.txt to the .rc file when building this plugin. If not, contact its maintainer.
```

FiveM commit:
https://github.com/citizenfx/fivem/commit/200916a479effdb3db586756e89fc916267c65ee

**Comment by Firecul** [comment]
Or just use my branch that has been kept up to date for years.
That method they described didn't work too well, the community @kibook found a better way to include compatibility.

**Comment by JamesSc0tt** [comment]
@Firecul - That would be good, but this is the one that is linked on the FiveM forums, so a lot of people will end up here :(

**Comment by Firecul** [comment]
> @Firecul - That would be good, but this is the one that is linked on the FiveM forums, so a lot of people will end up here :(

Yeah, I was posting mine for a while but I gave up when I lost interest. I still keep up compatibility and vehicles but I haven't personally updated the weapons in ages, I've pulled those updates
